services/grip/frontend/src/components/common/QRCode.vue

```diff
@@ -26,10 +25,15 @@ const copy_to_clipboard = (content: string) => {
       :size="400"
       level="H"
     />
-    <Accordion>
+    <Button
+      class="clipboard-button"
+      label="Copy to Clipboard"
+      icon="pi pi-paperclip"
+      @click="copy_to_clipboard(qr_content)"
+    ></Button>
+    <Accordion class="qr-accordion">
       <AccordionTab header="View Raw Content">
-        <i @click="copy_to_clipboard(qr_content)" class="pi pi-paperclip"></i>
-        <!-- {{ qr_content }} -->
+        {{ qr_content }}
```

Comment:
Can add a paragraph or span here with a class and then style that class with "word-wrap: break-word;"

Reply:
worked like a charm, thanks